cmd/deploy.go

```diff
@@ -49,9 +49,16 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var deployCmd = &cobra.Command{
 		Use:   "deploy",
-		Short: "Build and deploy your Appsody project to your Kubernetes cluster",
-		Long: `This command extracts the code from your project, builds a local Docker image for deployment,
-generates a deployment manifest (yaml) file if one is not present, and uses it to deploy your image to a Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
+		Short: "Build and deploy your Appsody project to Kubernetes.",
+		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
+		
+The command extracts the stack, along with your Appsody project to a local directory. The extracted project is used to build the container image for deployment using 'appsody build'.
+A deployment manifest file, "app-deploy.yaml", is generated if one is not present, and applied to the cluster. The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody Operator, and optionally as a Knative service if you specify the "--knative" flag.`,
```

Comment:
After this line, I would add something like:
`If the Appsody Operator cannot be found, it will be installed on your cluster.`